javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -300,6 +300,20 @@ public static boolean areSamePrimitiveTypes(TypeMirror left, TypeMirror right) {
         return (left.getKind() == right.getKind());
     }
 
+    /**
+     * Returns true iff the arguments are type variables with the same name. This method doesn't
+     * distinguish the difference of type variables having the same name but declared in different
+     * contexts, so we should be careful using it. Javac is able to identify the difference and
+     * issue an unchecked warning for the above scenario.
+     *
+     * @param v1 a type variable
+     * @param v2 a type variable
+     * @return whether the arguments are the same type variables
+     */
+    public static boolean areSameTypeVariables(TypeVariable v1, TypeVariable v2) {
+        return v1.asElement().getSimpleName() == v2.asElement().getSimpleName();
```

Comment:
This seems really undesirable. Two different type variable that have the same simple name "T" are not the same. Can you think of better logic for the single use of this function?